components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Stack/Models/EnumDefinition2.cs

```diff
@@ -25,11 +25,6 @@ public class EnumDefinition2 : EnumDefinition {
         /// </summary>
         public LocalizedText Description { get; internal set; }
 
-        /// <summary>
-        /// Whether the enum is an option set
-        /// </summary>
-        public bool IsOptionSet { get; set; }
```

Comment:
>IsOptionSet [](http://example.com/codeflow?start=20&length=11)

ist this already defined in the base class?

Reply:
Yes.
![image](https://user-images.githubusercontent.com/5628905/119346028-f4528900-bc99-11eb-919b-dd0fd4a0fcad.png)